### DISCUSSION TITLE
new version

### DISCUSSION THREAD

**Discussion by sidielnikov** [samuel-lucas6/Kryptor#50] [General]
Hello, when a new version is kryptor?

**Comment by samuel-lucas6** [comment]
Afraid I'm not sure when one will come out. I've reached the busiest time of the academic year, which involves working on multiple pieces of coursework at once and a dissertation. I'm currently focusing my programming efforts on rewriting all the tests in [Geralt](https://github.com/samuel-lucas6/Geralt), which is used in Kryptor. I'm also adding support for a new algorithm but need to figure out the API.

The Kryptor [roadmap](https://www.kryptor.co.uk/roadmap) was updated the other day, and I intend to get a patch release out before working on anything more substantial.

The issue is most of the bigger things I'd adjust would require breaking changes, which I'm reluctant to do. One non-breaking change idea is support for YubiKeys (#38), which makes sense if passkeys take off. However, that's not the easiest thing and shouldn't be expected any time soon. Putting a Kryptor private key on a USB stick already works and is good enough for all intensive purposes because it's encrypted.

**Reply by sidielnikov** [reply]
I really like the topic with yubikey
